src/components/inline-inputs/inline-inputs.style.js

```diff
@@ -20,83 +33,25 @@ const StyledInlineInputs = styled.div`
     width: 30px;
   }
 
-  .carbon-row {
+  ${StyledRow} {
     flex-grow: 1;
   }
 
-  .carbon-column + .carbon-column {
+  ${StyledColumn} + ${StyledColumn} {
     margin-left: -1px;
   }
 
-  .carbon-row.carbon-row--gutter-extra-small {
-    margin-bottom: 0;
-    margin-left: -8px;
-
-    > .carbon-column {
+  ${({ gutter }) => css`
+    ${StyledRow} {
       margin-bottom: 0;
-      padding-left: 8px;
-    }
-  }
-
-  .carbon-row.carbon-row--gutter-small {
-    margin-bottom: 0;
-    margin-left: -16px;
+      margin-left: -${spacings[gutter]}px;
```

Comment:
Could gutter not equal "none" which would cause some invalid CSS in `InlineInputs` or `Row`?